docs/infrastructure/koji-user-management.md

```diff
@@ -0,0 +1,168 @@
+  it _is_ enough to let them do scratch builds.
+- **software-contributor**: the ability to build into _any_ of the development repos _and_ promote into the osg-testing repos (but not any other repos).
+  Should be given to people that build specific software but do not belong to the OSG S&R team or HCC.
+- **operations-team**: the ability to build into development and promote into testing the `vo-client` package.
+  Should be given to people in OSG Operations.
+- **security-team**: the ability to build into development and promote into testing the `*-ca-certs*` packages.
+  Should be given to people in OSG Security.
+- **hcc-build**: the ability to build and promote into any of the `hcc` tags.
+  Should be given to people in Nebraska's Holland Computing Center.
+- **software-team**, **release-team**: the ability to build any package into any development tag, and promote to any tag.
+  Current policy does not distinguish between these two permissions.
+  Should be given to people on the S&R team.
+- **admin**: the ability to do anything (short of direct database or config manipulation).
+  Should be given sparingly.
+
+The `grant-permission` command can also be used to create new permissions; doing so is beyond the scope of this document.
+For further permission details, see the [policy writing doc](https://opensciencegrid.github.io/technology/infrastructure/koji-policy-writing/)
+and `/etc/koji-hub/hub.conf` on koji.chtc.wisc.edu.
+
+
+Handling User Cert Changes / Renaming a User
+--------------------------------------------
+
+Koji users are identified by the _first_ Common Name (CN) of their X.509 certificate.
+If this changes for a user (e.g. they get a cert from a new CA), someone with root access on koji.chtc.wisc.edu will
+need to SSH there and do the following:
+
+``` console
+$ sudo /root/psql-kojidb
+```
+``` psql
+koji=> BEGIN;
+koji=> SELECT * FROM users WHERE name='<OLDNAME>';
+-- make sure you find one and exactly one person
+koji=> UPDATE users SET name='<NEWNAME>' WHERE name='<OLDNAME>';
+-- 1 row should have been updated
+koji=> SELECT * FROM users;
+-- sanity check; if everything looks ok, commit
+koji=> COMMIT;
+koji=> \q
+```
+If you mess up, do `ROLLBACK; BEGIN;` and try again.
+
+Inform the user:
+
+- they can no longer use their old cert
+- if they _aren't_ using a proxy for Koji auth, they must rerun `osg-koji setup` to fix their `client.crt` files
+- they must import their new cert into their browsers
+- they must clear their browser cache and cookies for koji.chtc.wisc.edu before using the web interface
+  (or else they'll get a Python stack trace when they try to connect)
+
+
+Disabling and Enabling a User
+-----------------------------
+
+Users cannot be deleted, but they can be disabled.
+
+To disable a user, use the command:
+``` console
+$ osg-koji disable-user "<USERNAME>"
+```
+If you are feeling extra paranoid, use `osg-koji revoke-permission` to remove all their permissions.
```

Comment:
We should give a clear decision point for an appropriate time to revoke permissions. I'm ok with just getting rid of this sentence since permission revocation is mentioned above.